### ISSUE TITLE
"." characters in path cause error

### ISSUE THREAD

**Issue by bernardb2010** [KosmosisDire/obsidian-webpage-export#90]
First, I'd like to say thanks for this awesome plugin. It has been a huge help.

**Describe the bug**
Trying to export a folder with notes.

Specific directory structure:

Vault Name       DIR            FILE
Notes          -> 2. Calls -> FILENAME.md

I can export FILENAME.md by itself, but every time I try to export "2. Calls", I get the error below.

This also occurred in another folder in the same vault. "3. Meetings".

This directory is structured as:
Vault Name       DIR                 SUB DIR                 FILE
Notes          -> 3. Meetings-> general      ->FILENAME.md

In the 3. Meetings directory, there are (4) sub directories. There are also notes in the "root" 3. Meetings folder. Trying to export the folder results in the same error below. If i move the notes from 3. Meetings directory into general directory, the 3. Meetings folder is successfully able to export.

Hopefully this explanation makes sense...

**Operating System**
Windows 10

**To Reproduce**
1. Right click on folder with notes
2. Export to html

##### [Error] Could not export file: FILENAME.md
```
Error: vaultRelativeDestination must be a folder
    at new Downloadable (plugin:webpage-html-export:10600:13)
    at ExportFile.getSelfDownloadable (plugin:webpage-html-export:12162:12)
    at HTMLGenerator.generateWebpage (plugin:webpage-html-export:11723:33)
    at async HTMLExportPlugin.exportFile (plugin:webpage-html-export:12306:7)
    at async HTMLExportPlugin.exportFolder (plugin:webpage-html-export:12346:28)
    at async eval (plugin:webpage-html-export:12268:30)
```

**Comment by KosmosisDire** [comment]
Huh interesting, I think this is due to the "." In your folder name, try exporting a folder with no "."s in their name or path and let me know if that works. Thanks!

**Comment by bernardb2010** [comment]
Removing the "." worked. Just for information, this was working in a previous version. Is this something that may be updated?

**Comment by KosmosisDire** [comment]
Yes, I rewrote the whole path system. I am honestly surprised there aren't **more** bugs. But yes, no worries I will fix this as it is 100% a bug.

**Comment by KosmosisDire** [comment]
This is fixed in update 1.6.0, let me know if you have any more problems! And thanks for the report :)
